### PR TITLE
Use deploy step instead after_success

### PR DESCRIPTION
**Pull request by CAMOBAP**
-

### CODE REVIEW

**Comment by ronaldtse** [comment]
@CAMOBAP795 excellent! Will merge once it succeeds!

**Comment by CAMOBAP** [comment]
After merge I will retag to make sure that `docker push` works

**Review by ronaldtse — APPROVED** [review]

**Comment by CAMOBAP** [comment]
retag in progress https://travis-ci.com/riboseinc/jenkins-docker/builds/106875863

**Pull request merged**
